### PR TITLE
Add voltage init parameters

### PR DESCRIPTION
**Pull request by Meklo**
### Note 
In order to be able to launch voltage init combined with voltage limits defined in the new parameter tab you have to ensure the targeted voltage levels of the selected filters has a numerical value assigned to lowvoltagelimit and highvoltagelimit in voltagelevel table of iidm schema.
If it is equal to NaN the process will crash

### CODE REVIEW

File: src/components/dialogs/parameters/parameters.js [hunk 1]
```diff
@@ -619,6 +633,21 @@ const Parameters = ({ user, isParametersOpen, hideParameters }) => {
                                 />
                             )}
                         </TabPanel>
+                        <TabPanel
```
Comment (etiennehomer) [inline src/components/dialogs/parameters/parameters.js:636]: It should be in dev mode only

**Review by ayolab — APPROVED** [review]

File: src/components/utils/rhf-inputs/table-inputs/table-numerical-input.js [hunk 2]
```diff
@@ -94,17 +96,24 @@ export const TableNumericalInput = ({
...
-                        <IconButton
-                            onClick={handleClearValue}
-                            style={{
-                                visibility: clearable ? 'visible' : 'hidden',
-                            }}
-                        >
-                            <ClearIcon />
-                        </IconButton>
-                    </InputAdornment>
+                    <>
```
Comment (etiennehomer) [inline src/components/utils/rhf-inputs/table-inputs/table-numerical-input.js:99]: Why do you add this level ?

File: src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.js [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+                arrayFormName={`${VOLTAGE_LIMITS}`}
+                columnsDefinition={COLUMNS_DEFINITIONS}
+                useFieldArrayOutput={useFieldArrayOutput}
+                createRows={createVoltageLimitRows}
+                tableHeight={270}
+                withAddRowsDialog={false}
+                withLeftButtons={false}
+            />
+
+            <DialogActions>
```
Comment (etiennehomer) [inline src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.js:250]: You should add a close button like other parameters tabs

Comment (Meklo) [reply src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.js:255]: agreed, it's done

File: src/components/utils/rhf-inputs/table-inputs/table-numerical-input.js (hunk 2 above)
Comment (Meklo) [reply src/components/utils/rhf-inputs/table-inputs/table-numerical-input.js]: It's actually unnecessary, i've removed it

File: src/components/dialogs/parameters/parameters.js (hunk 1 above)
Comment (Meklo) [reply src/components/dialogs/parameters/parameters.js:638]: alright, it's done

**Review by etiennehomer — APPROVED** [review]

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**